Source/WebCore/rendering/RenderObject.h

```diff
@@ -269,7 +269,7 @@ class RenderObject : public CachedImageClient {
     virtual bool isRenderScrollbarPart() const { return false; }
     virtual bool isRenderVTTCue() const { return false; }
 
-    bool isDocumentElementRenderer() const { return document().documentElement() == m_node; }
+    bool isDocumentElementRenderer() const { return document().documentElement() == &m_node.get(); }
```

Comment:
This could be `== m_node.ptr()`, I think?

Reply:
Yes, will fix. I didn't realize we had `ptr()` on CheckedRef.